cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -304,6 +307,22 @@ rules:
   - list
   - watch
 ---
+# Aspects of Rook Ceph Agent that require access to secrets
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
```

Comment:
i'm concerned about always creating this clusterrole, even if there is no binding by default. Could you add this to the documentation where the binding is created?

Reply:
@travisn Why are you concerned about that? The ClusterRole will just exist and do nothing unless it is bound using a ClusterRoleBinding or RoleBinding.
From my view makes it easier for people to upgrade as it is in the `operator.yaml`.

Reply:
If it's only needed for this scenario and they need to create other yaml already, why not delay creating it? It gives the appearance that the operator needs access to secrets in all namespaces. But of course you're right it doesn't change the behavior of the system.

Reply:
I moved it out. But for the helm chart if the mount security mode is set to "anything" else then `Any` the ClusterRole will be created to have it convenient for Helm users.